src/shims/env.rs

```diff
@@ -109,3 +109,10 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         }
     }
 }
+
+pub(crate) fn get_pid<'tcx>(ecx: &InterpCx<'tcx, MiriMachine<'tcx>>, item_name: &str) -> u32 {
```

Comment:
Please don't use free functions when this can easily become a method on the extension trait.

Reply:
done.

Is there a reason methods in extension trait are preferred? To not pollute the namespace?

Reply:
To avoid having to import each function separately, and to keep things consistent.